recipes/yaml

```diff
@@ -0,0 +1,3 @@
+(yaml :repo "zkry/yaml.el"
+      :fetcher github
+      :files ("yaml.el"))
```

Comment:
You can, and should, omit the `:files` spec here since the defaults do what you want, ie. it excludes the test file automatically. :-)

Reply:
Thanks for the suggestions! I've updated the repos. I do think using `pcase` looks better.

Reply:
Cool, you should use `_` as the catch-all case at the end of the file now instead of `t` btw -- flycheck showed me this when I looked at the new code locally.

Reply:
Also some of the other `cond`s can potentially use pcase too, because I _think_ it knows about string comparisons, e.g. for
```
                    (cond ((equal c "block-key") "flow-key")
                          ((equal c "flow-in") "flow-in")
                          ((equal c "flow-key") "flow-key")
                          ((equal c "flow-out") "flow-in"))
```

Reply:
nice catch :)